0-ad-history-mod-package/simulation/templates/structures/han/civil_centre.xml

```diff
@@ -10,6 +10,12 @@
   <Identity>
     <Civ>han</Civ>
     <SpecificName>Guān Shǔ</SpecificName>
+    <History>
+         The Han empire was absolutely governed by an emperor who would pass on the throne to his eldest son. He was the supreme lawgiver, chief commander of the military, and designator of senior government officials. However, the emperor was assisted by a complex government system. It consisted of numerous layers and institutions and was meant to simplify political procedures without letting anyone (except the emperor) amass power. The system was structured as follows (simplified):
+         At the top were three officials (called The Three Excellencies), each responsible for a different domain: The Chancellor stood above the population and the lands and was responsible for the state's finances. The Grand Commandant held command over the empire's military. The Imperial Secretary had the duty of overseeing and monitoring the government and its officials.
```

Comment:
For this paragraph, I think it would be better if it was like this:
```suggestion
         At the top were three officials: the Chancellor, The Grand Commandment, and the Imperial Secretary. These three were collectively called the Three Excellencies; each were responsible for a different domain: The Chancellor stood above the population and the lands and was responsible for the state's finances. The Grand Commandant held command over the empire's military. The Imperial Secretary had the duty of overseeing and monitoring the government and its officials.
```
The paragraph flows better if we are introduced to the Three Excellence's early and then list their tasks.